docs/source/analyze/weave-report.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Weave Report
+description: Analyze weave reports
+---
+
+The weave analyzer runs several checks to detect problems with the [weave](https://www.weave.works/docs/net/latest/kubernetes/kube-addon/) CNI provider.
```

Comment:
```suggestion
The Weave analyzer runs several checks to detect problems with the [Weave](https://www.weave.works/docs/net/latest/kubernetes/kube-addon/) Container Network Interface (CNI) provider.
```
Capitalization and avoiding first use of an acronym (presumed we should follow similar guidelines as the [kots contributions guidelines](https://github.com/replicatedhq/kots.io/blob/master/CONTRIBUTING.md))